modules/installation-vsphere-config-yaml.adoc

```diff
@@ -116,114 +111,115 @@ imageContentSources: <17>
+// password
 <11> The password associated with the vSphere user.
+// server
 <12> The fully-qualified hostname or IP address of the vCenter server.
 +
 [IMPORTANT]
 ====
 The Cluster Cloud Controller Manager Operator performs a connectivity check on a provided hostname or IP address. Ensure that you specify a hostname or an IP address to a reachable vCenter server. If you provide metadata to a non-existent vCenter server, installation of the cluster fails at the bootstrap stage.
 ====
+// diskType
 <13> The vSphere disk provisioning method.
 ifndef::openshift-origin[]
+// fips
 <14> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
 +
 [IMPORTANT]
 ====
 To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base-full} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on RHEL, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode]. When running {op-system-base-full} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.
 ====
 endif::openshift-origin[]
-ifndef::restricted[]
 ifndef::openshift-origin[]
+ifndef::restricted[]
+// pullSecret
 <15> The pull secret that you obtained from {cluster-manager-url}. This pull secret allows you to authenticate with the services that are provided by the included authorities, including Quay.io, which serves the container images for {product-title} components.
+// sshKey
 <16> The public portion of the default SSH key for the `core` user in
 {op-system-first}.
-endif::openshift-origin[]
-ifdef::openshift-origin[]
-<15> You obtained the {cluster-manager-url-pull}. This pull secret allows you to authenticate with the services that are provided by the included authorities, including Quay.io, which serves the container images for {product-title} components.
-<16> The public portion of the default SSH key for the `core` user in
-{op-system-first}.
-+
-[NOTE]
-====
-For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
-====
-endif::openshift-origin[]
 endif::restricted[]
+endif::openshift-origin[]
+////
```

Comment:
An error exists with callouts from this point forward. 

Logic is not making sense:

asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 172: callout list item index: expected 14, got 15
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 177: callout list item index: expected 15, got 16
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 185: callout list item index: expected 16, got 17
asciidoctor: WARNING: modules/installation-vsphere-config-yaml.adoc: line 188: callout list item index: expected 17, got 18